app/context/userContext.tsx

```diff
@@ -354,17 +354,6 @@ export const UserContextProvider = ({ children }: { children: any }) => {
       setSigner(new Web3Provider(wallet.provider).getSigner());
       // Login to Ceramic
       passportLogin();
-      // attach listener
-      wallet.provider.on("chainChanged", async (chainId: string): Promise<void> => {
```

Comment:
@nutrina I remembered at one point we ran into a jwt issue relating to switching networks on the scorer. I needed to remove this for the on chain functionality and everything seems good to me but just wanted to flag it just in case